generic-desktop-kde-live-20160731.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "generic-desktop"

```

Comment:
maybe you can help with https://progress.opensuse.org/issues/12956#open-questions